scripts/postinstall.js

```diff
@@ -20,13 +22,17 @@ try {
 
 function setupAutocomplete() {
   return new Promise((resolve, reject) => {
+    let tabtabPath = './node_modules/tabtab';
+    if (!fileExistsSync(path.join(__dirname, '..', 'node_modules', 'tabtab', 'src', 'cli.js'))) {
+      tabtabPath = '../tabtab';
```

Comment:
I thought so. ( and at first, I used get-installed-path. )
But,  "get-installed-path" did not work, when I excute `npm install serverless --save`.